app/controllers/applies_controller.rb

```diff
@@ -19,7 +22,7 @@ def approve
   def deny
     @mission = Mission.find(params[:id])
     @mission.status = "denied"
-    @mission.save
+    @mission.save!
 
     authorize @mission
     redirect_to mentor_profil_path
```

Comment:
Wondering if the approve or deny shouldn't be in the mission crontroller since we're only manipulation `@missions`